lua/carrier/packages/carrier/_ctor.lua

```diff
@@ -36,6 +36,7 @@ end
 
 local debugColor = Color (192, 192, 192)
 function Carrier.Debug (message)
+	if Carrier.IsLocalDeveloperEnabled and not Carrier.IsLocalDeveloperEnabled () then return end
```

Comment:
Carrier.IsLocalDeveloperEnabled is defined in developer.lua

Reply:
Are we guaranteed to have IsLocalDeveloperEnabled by loading order?

Reply:
Yes